packages/paste-core/components/datagrid/src/utils/modded-reakit.ts

```diff
@@ -0,0 +1,158 @@
+/**
+ * Lightly modified functions copied from Reakit
+ * https://github.com/reakit/reakit/tree/master/packages/reakit-utils/src
+ * Used for focus management and tabbable controls in the datagrid
+ */
```

Comment:
I'm not sure I saw them but it might be useful have some commentary on what we lightly modified.

Reply:
Also can we steal any tests reakit had for these for our test suite? Just for coverage and safety